manifests/conf.pp

```diff
@@ -40,6 +40,7 @@
   $content          = undef,
   $source           = undef,
   $template         = undef,
+  $template_epp     = undef,
```

Comment:
Add a type:
```
Optional[Hash[String[1], Hash]] $template = undef,
```
This will also make validation easier.

Reply:
I tried your type suggestion but couldn't get it to work. It all looks right to me. But I just can't get it.

![image](https://github.com/saz/puppet-sudo/assets/20624453/7e96c4cd-96c6-488d-85fd-95cc7a322869)
![image](https://github.com/saz/puppet-sudo/assets/20624453/1e8b2eb8-4a42-46de-a18b-84cf28c4b0f0)
![image](https://github.com/saz/puppet-sudo/assets/20624453/bbc59cb8-a313-492a-9b24-8565567f82ba)

I'd welcome any suggestions you might have

Reply:
btw: it would be good to have some tests for this functionality

Reply:
I tried your new type and it worked perfectly. Thanks for that. I definitely would not have ever come up with that.